### PR TITLE
2111.1 Download Page updates

### PR DESCRIPTION
**Pull request by erickson-doug**
Signed-off-by: Doug Erickson <dougeric@amazon.com>

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Review by AMZN-daimini — APPROVED** [review]

File: layouts/partials/download/content-linux.html [hunk 1]
```diff
@@ -3,35 +3,38 @@
...
         <p id="github-page-button-windows">
-          <a href="#" class="btn-xl-block btn-lg btn-green text-white text-center" target="_blank">Fork from GitHub</a>
+          <a href="https://github.com/o3de/o3de/main" class="btn-xl-block btn-lg btn-green text-white text-center" target="_blank">Fork from GitHub</a>
         </p>
       </div>
     </div>
     <div class="col-12 col-lg-7">
       <h5>Minimum System Requirements</h5>
-      <p>O3DE requires Windows 10 64-bit (versions 1809, 10.0.17763 or later)</p>
+      <p>O3DE (for Linux) requires Ubuntu 18.04 LTS.</p>
```
Comment (willihay) [inline layouts/partials/download/content-linux.html:20]:
```suggestion
      <p>O3DE for Linux requires Ubuntu 20.04 LTS.</p>
```

Comment (willihay) [reply layouts/partials/download/content-linux.html:20]: Ubuntu version requirements were recently updated.

File: layouts/partials/download/content-linux.html [hunk 2]
```diff
@@ -3,35 +3,38 @@
...
         </ul>
-        <li>60 GB of free disk space</li>
-        <li>Visual Studio 2019</li>
+        <li>40 to 100 GB of free disk space</li>
+        <li>1366 x 768 px screen resolution</li>
       </ul>
+      <p> For more detailed requirements, read <a href="https://o3de.org/docs/welcome-guide/requirements/">O3DE System Requirements</a>.</p>
       <h5>Instructions</h5>
-      <p>Choose the "Download O3DE" button, and read the documentation and follow the instructions in it.</p>
+      <p>Choose the "Download O3DE" button, <a href="https://o3de.org](/docs/welcome-guide/setup/installing-linux/">read the documentation</a>, and follow the instructions in it.</p>
```
Comment (willihay) [inline layouts/partials/download/content-linux.html:37]:
```suggestion
      <p>Choose the "Download O3DE" button, <a href="https://o3de.org/docs/welcome-guide/setup/installing-linux/">read the documentation</a>, and follow the instructions in it.</p>
```

File: layouts/partials/download/getting-started-linux.html [hunk 3]
```diff
@@ -24,10 +24,10 @@ <h4>Setting up O3DE from GitHub</h4>
         </a>
       </div>
       <div class="col col-12 col-md-6 col-lg-3">
-        <a href="https://o3de.org/docs/welcome-guide/create/creating-projects-using-cli/">
+        <a href="https://o3de.org](/docs/welcome-guide/setup/installing-linux/">
           <div class="card mb-3">
             <h4>Install O3DE via Debian Package</h4>
-            <p>Lorem Ipsum Dolor Sit Amet.</p>
+            <p>Learn how to install O3DE for Linux (Ubuntu 18.04 LTS).</p>
```
Comment (willihay) [inline layouts/partials/download/getting-started-linux.html:30]:
```suggestion
            <p>Learn how to install O3DE for Linux (Ubuntu 20.04 LTS).</p>
```

**Review by willihay — CHANGES_REQUESTED** [review]

**Pull request merged**